packages/button/src/react/index.tsx

```diff
@@ -191,16 +214,19 @@ const Button = React.forwardRef<ButtonElement, ButtonProps>(
 
     const themeName = useTheme()
 
-    const ref = React.useRef<HTMLAnchorElement | HTMLButtonElement>()
-    React.useImperativeHandle(forwardedRef, () => ref.current)
+    const ref = React.useRef<HTMLButtonElement>()
+    React.useImperativeHandle(
+      forwardedRef,
+      () => (ref.current as unknown) as HTMLButtonElement
```

Comment:
If it can be removed without throwing an error I'm for it.

Reply:
Great, we are agreed. Do you plan to try removing it?

Reply:
Removing it throws the following 

Type 'HTMLButtonElement | undefined' is not assignable to type 'HTMLButtonElement'.
  Type 'undefined' is not assignable to type 'HTMLButtonElement'.ts(2322)
index.d.ts(1095, 79): The expected type comes from the return type of this signature.
(property) MutableRefObject<HTMLButtonElement | undefined>.current: HTMLButtonElement | undefined

---
Currently we cast like so in the following
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/dropdown/src/react/item.tsx#L35-L40
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/emptystate/src/react/index.tsx#L88-L90
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/errors/src/react/error-page.tsx#L93-L95
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/focusmanager/src/react/index.tsx#L14-L16
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/position/src/react/index.tsx#L103-L105
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/radio/src/react/button.tsx#L63-L66
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/scrollable/src/react/index.tsx#L55-L57
https://github.com/pluralsight/design-system/blob/15887e8fe9de05d2aef21cdf7240dc3097f50472/packages/switch/src/react/index.tsx#L132-L137

Do you want to create card to figure out a way to remove all these as a separate issue? This popped up in button now because of the `strict:true`.